plugins/amazonq/codewhisperer/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codewhisperer/popup/QInlineCompletionProvider.kt

```diff
@@ -592,7 +592,7 @@ class QInlineCompletionProvider(private val cs: CoroutineScope) : InlineCompleti
         val project = editor.project ?: return false
 
         if (!isQConnected(project)) return false
-        if (!CodeWhispererExplorerActionManager.getInstance().isAutoEnabled() && event.isManualCall()) return false
+        if (!CodeWhispererExplorerActionManager.getInstance().isAutoEnabled()) return false
```

Comment:
won't this will disable completions completely if auto is not on

Reply:
```kt
// not sure if current impl is completely correct either
if (event.isManualCall()) return true
if (!CodeWhispererExplorerActionManager.getInstance().isAutoEnabled()) return false
```

Reply:
We want it to disable when auto is not on right? Auto is the pause and resume suggestions in the dropdown

Reply:
Discussed offline, enabling manual call for when auto-suggestions are paused